src/librustc_mir/shim.rs

```diff
@@ -39,8 +39,6 @@ fn make_shim<'tcx>(tcx: TyCtxt<'tcx>, instance: ty::InstanceDef<'tcx>) -> Body<'
             // FIXME(eddyb) support generating shims for a "shallow type",
             // e.g. `Foo<_>` or `[_]` instead of requiring a fully monomorphic
             // `Foo<Bar>` or `[String]` etc.
-            assert!(!ty.needs_subst());
-
```

Comment:
Did you mean to remove the comment like you did below?

Reply:
Yes, fixed!